### ISSUE TITLE
Create custom metric to log accurcy per class 

### ISSUE THREAD

**Issue by jkbdnj** [jkbdnj/jakub-dunaj-bachelors-thesis#20]
### Task description 

Create custom metric to log accurcy per class after each epoch/training run.

[base_class_metric_keras](https://keras.io/api/metrics/base_metric/)
[base_metric_source_code](https://github.com/keras-team/keras/blob/v3.6.0/keras/src/metrics/metric.py)

**Comment by jkbdnj** [comment]
*NOTE:*
from [https://keras.io/api/metrics/base_metric/#metric-class](https://keras.io/api/metrics/base_metric/#metric-class)
<img width="706" alt="metrics" src="https://github.com/user-attachments/assets/39a568bc-0cd0-4141-974f-edeb2ef23013">
